services/config/field_maps.py

```diff
@@ -41,6 +41,16 @@ def handle_strip(value):
         return value
 
 
+def handle_no_choice_to_empty_string(value):
+    """
+    Useful for going from a single choice field to a short text datatype.
+    If no option is picked for a single choice the API will return None.
+    The short text field will return an empty string.
+    """
+    if not value:
+        return ""
+
+
```

Comment:
datatypes in knack are weird

Reply:
this is annoying. i guess Knackpy can't really handle this automatically, because maybe someone really does want an empty string instead of a null value 🙃